### ISSUE TITLE
screen is completely not viewable

### ISSUE THREAD

**Issue by WholesomeIsland** [calc84maniac/tiboyse#2]
![rn_image_picker_lib_temp_71d3d24f-b5da-463f-be86-deaf13fe3974](https://user-images.githubusercontent.com/65096556/190725059-5f1ab469-dbf3-455c-bd1c-2a21f96204de.jpg)

pic attached, the screen is looking very strange, this is the title screen of the bundled game

**Comment by calc84maniac** [comment]
I got another report about this via email about a month ago, it seems to be related to a change in the LCD controller that makes the old method of ready-waiting cause corruption in the output. I should be able to address this at some point, though right now I'm focused on TI-Boy CE development.

**Comment by WholesomeIsland** [comment]
I didn't know if you were active at all so I had also sent u an email, thanks for letting me know what the problem was.

**Comment by Grantrocks** [comment]
Is there ever a chance in the future that this will be fixed?

**Comment by calc84maniac** [comment]
I was able to purchase a newer TI-84+ that exhibits the problem, and implemented a proof-of-concept fix a few months ago. I could put some more focus on making a more proper fix sometime soon (that works on both old and new hardware).

**Comment by Grantrocks** [comment]
That would be awesome!On Feb 6, 2024 1:11 PM, calc84maniac ***@***.***> wrote:
I was able to purchase a newer TI-84+ that exhibits the problem, and implemented a proof-of-concept fix a few months ago. I could put some more focus on making a more proper fix sometime soon (that works on both old and new hardware).

**Comment by personthatl** [comment]
I am seeing this problem as well and i cannot play anything regardless of how much space it takes up.

**Comment by Delmirev-Arjhan** [comment]
I am also having this issue. I've reset the OS, redownloaded the game (tetris) onto the calculator, re translated the file from the ROM, and swapped the batteries. idk what else to do or if I can do anything

**Comment by ifonlymycalcplayedGB** [comment]
same here. Tried both the 0.2.05 app and 0.2.04 app; neither worked. I have a VERY new ti-84 plus so that might be why. Should I watch for a fix soon?

**Comment by calc84maniac** [comment]
This issue should now be fixed by the new release: https://github.com/calc84maniac/tiboyse/releases/tag/0.2.06

**Comment by karsonburchett74-hash** [comment]
Yeah the new version does not have an "Adjstris" file to run the ROMs. Same thing happens when i boot it up because i have to use the old app.

**Comment by karsonburchett74-hash** [comment]
All there is to use in the file is a make-app file[

<img width="868" height="200" alt="Image" src="https://github.com/user-attachments/assets/c29b5214-203b-4972-9c7c-eb4982619b65" />

](url)

**Comment by karsonburchett74-hash** [comment]
nevermind i figured it out, just click on the game in apps not the adjstris file
